### ISSUE TITLE
Crossterm backend goes inverted on progress bar

### ISSUE THREAD

**Issue by tenten8401** [gyscos/Cursive#339]
# Problem description
When using the crossterm backend, displaying a progress bar on Linux will flip the terminal to a different color scheme, it does't happen with termion.

Crossterm:
![Screenshot_20190428_141834](https://user-images.githubusercontent.com/6174343/56868446-86570d00-69c0-11e9-84cf-93c9ebf88ab8.png)

Termion:
![Screenshot_20190428_141759](https://user-images.githubusercontent.com/6174343/56868439-73dcd380-69c0-11e9-95c9-a02c71783ca3.png)

# Environment

* Operating system: Fedora 30 KDE
* Backend used: crossterm
* Current locale: en_US.UTF-8
* Cursive version: 0.11.2

**Comment by gyscos** [comment]
Indeed! It seems to also trigger with the (simpler) `edit` example, though I can't find so far what's causing it. Most of the logic to handle colors was copied from the termion backend, so it might either be a problem in the higher-level code (either `Printer`, `Cursive`, or some intermediate view implementation) that happened to be fine with the current backends, or some bug in crossterm itself (or in the way the backend uses it).

I'll investigate more when I get some available time.

**Comment by vmedea** [comment]
You might want to try again, it seems here that this is fixed by #399.
